deploy/helm/sumologic/values.yaml

```diff
@@ -390,6 +412,52 @@ fluentd:
     ## To use additional output plugins
     extraOutputPluginConf: |-
 
+    ## Output configuration
+    ## match: fluentd match
```

Comment:
nit - we should call this tag imo because that is what it really is, the tag for the match block.

Reply:
fixed